index.html

```diff
@@ -24,7 +24,7 @@
 				<li><a href="#getting-started">Getting Started</a></li>
 				<li><a href="#examples">Examples</a></li>
 				<li><a href="#api">API</a></li>
-				<li><a href="#support">Browser support</a></li>
+				<li><a href="#support">Browser Support</a></li>
```

Comment:
I don't quite agree with this. What's the reasoning behind capitalizing support?